scripts/docker_manager_upgrade.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+fork do
```

Comment:
can you add a comment explaining why the double forking is done, can be in another PR since I merged this already

Reply:
did we need the   `Process.setsid` thing? I think we lost it.